mkdocs.yml

```diff
@@ -1,16 +1,46 @@
 site_name: Armbian Documentation
+site_author: "Armbian team"
+
+copyright: Copyright &copy; 2015 - 2022 Armbian team
 repo_url: https://github.com/armbian/documentation
-repo_name: Github
+repo_name: armbian/documentation
 
-site_author: "Armbian team"
-google_analytics: ["UA-284946-9", "docs.armbian.com"]
+theme: 
+    name: material
+    custom_dir: overrides
+    logo: images/logo.svg
+    icon:
+      repo: fontawesome/brands/github-alt
+    palette:
+        - scheme: default
+          primary: red
+          accent: red
+          toggle:
+            icon: material/toggle-switch-off-outline 
+            name: Switch to dark mode
+        - scheme: slate
+          primary: red
+          accent: red
+          toggle:
+            icon: material/toggle-switch
+            name: Switch to light mode
+    favicon: images/logo.png
+    features:
+        - navigation.tabs
+        - navigation.top
+        - toc.integrate
 
-theme:
-  name: readthedocs
-  highlightjs: true # enable syntax highlight for code blocks
+extra:
+  analytics:
+    provider: google
```

Comment:
What is this for?

Reply:
for theme intergrated google analyze code extension

Reply:
UA-284946-9 is for docs.armbian.com

Reply:
ok